src/settings.json

```diff
@@ -132,5 +133,19 @@
     "1": {"setting_name": "shutter_a", "gpio_pin": 16},
     "2": {"setting_name": "fps", "gpio_pin": 26},
     "3": {"setting_name": "wb", "gpio_pin": 5}
+  },
+  "i2c_oled": {
+    "enabled": true,
```

Comment:
enabled is the only required settings since all other settings have default fallbacks.